train/output/metrics_bert-base-multilingual-cased.md

```
@@ -0,0 +1,264 @@
+| Throughput | 22934 sps |
+
+## Overall evaluation
+
+| Metric | Value |
+|---|---|
+| Accuracy | 0.9455 |
+| Weighted F1 | 0.9446 |
+
+### Classification report
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)